test/models/subnet_test.rb

```diff
@@ -145,6 +146,9 @@ class Subnet::Test < Subnet; end
     assert_equal 1500, Subnet.new.mtu
   end
 
+  test "should have nic_delay set to nil by default" do
+    assert_equal nil, Subnet.new.nic_delay
+
```

Comment:
`end`? :)

Reply:
yark, that will teach me to do two things at once...